pkgs/development/python-modules/chevron/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "chevron";
+  version = "0.13.1";
+
+  src = fetchPypi {
```

Comment:
No tests distributed on `pypi`.

Can you use `fetchFromGitHub` to get an archive with tests and enable them?

Reply:
Fixed in cd688755f84ee4c9da816d2144a7d1eeb901086f.

Tests ran successfully.